### ISSUE TITLE
dependency error in Spring boot

### ISSUE THREAD

**Issue by mdiu2203** [jwtk/jjwt#669]
Hello, I'm grateful thanks to this library.
but I detected an error using this library in Spring Boot.
maybe it's because of Gradle version. I checked the error message as below.

```
Could not find method compile() for arguments [io.jsonwebtoken:jjwt-api:0.11.2] on object of type org.gradle.api.internal.artifacts.dsl.dependencies.DefaultDependencyHandler
```

It works under version 7. but not works more than version 7.
i think it's because of 'compile'.

**Comment by lhazlewood** [comment]
Hello there!

There is no `compile` method anywhere in the JJWT API, so I don't know what could be causing this.

Do you have a code sample/snippet or example project that demonstrates the behavior so we can troubleshoot?

**Comment by bdemers** [comment]
The ‘compile’ scope has been removed in Gradle v7. You must use ‘implementation’ now instead.

**Comment by lhazlewood** [comment]
@bdemers doesn't that mean we should update the docs here then? https://github.com/jwtk/jjwt#gradle

**Comment by bdemers** [comment]
Yup, sorry we had a related PR were we had talked about this. #664 

let’s reopen this one to keep track of it

**Comment by bdemers** [comment]
Fixed in: #665
